Feature/href.ve The driver script file names have been changed

BinbinZhou-NOAA opened the pull request:
## Pull Request Testing ##

- [y ] Describe testing already performed for this Pull Request:</br>

- [y ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ y] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ y] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [y ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [y ] Review the source issue metadata (required labels, projects, and milestone).
- [ y] Complete the PR description above.
- [y ] Ensure the PR title matches the feature branch name.
- [y ] Check the following:
- [ y]  Instructions provided on how to run
- [y ]  Developer's name is replaced by ${user} where necessary throughout the code
- [y ]  Check that the ecf file has all the proper definitions of variables
- [ y]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [y ]  Check to see that the output directory structure is followed
- [y ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ y] After submitting the PR, select **Development** issue with the original issue number.
- [y ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [y ] Close the linked issue.

BinbinZhou-NOAA committed:
updateplot scripts to modify the plot names

BinbinZhou-NOAA committed:
update modulefiles/cam/cam_plots.sh

BinbinZhou-NOAA committed:
update modulefiles/cam/cam_stats.sh

BinbinZhou-NOAA committed:
update plot scripts for graphic file name change

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/href.ve

BinbinZhou-NOAA committed:
change the spatial map file name to match naming standard

BinbinZhou-NOAA committed:
correct cpu request in spatial map job

BinbinZhou-NOAA committed:
update scripts/cam/plots/exevs_href_precip_spatial_plots.sh again

BinbinZhou-NOAA committed:
relocate COMROOT definition in spatial map job

BinbinZhou-NOAA committed:
update JEVS_CAM_STATS and v1.0 file

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/href.ve

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/href.ve

BinbinZhou-NOAA committed:
update machine.conf

BinbinZhou-NOAA committed:
remove line evs_ver=1.0 in all driver scripts

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/href.ve

BinbinZhou-NOAA committed:
change driver script file names

BinbinZhou-NOAA committed:
deleted useless driver script files

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/href.ve

BinbinZhou-NOAA commented:
3 changes: 
Changed driver script names from jevs_href_* to jevs_cam_href_* 
Removed evs_ver=1.0 from the driver scripts
removed  export MET_bin_exec=bin from the driver scripts

LoganDawson-NOAA commented:
@BinbinZhou-NOAA just attaching the request I sent via email to this PR.

We need to have your HREF grid2obs stats jobs updated to use BILIN interpolation so your stats can be included on the CAM headline plots being generated by @MarcelCaron-NOAA. Right now, your HREF stats are being filtered out because the INTERP_MTHD in your stats files is set to NEAREST while other EVS grid2obs stats are using BILIN for the INTERP_MTHD.

Will you update your METplus configuration files like below (as appropriate)?

{ENSEMBLE/POINT}_STAT_INTERP_TYPE_METHOD = BILIN
{ENSEMBLE/POINT} _STAT_INTERP_TYPE_WIDTH = 4

You'll also need to update the INTERP_MTHD that's set in the scripts that generate your HREF grid2obs.

BinbinZhou-NOAA commented:
Logan,

  Sure I'll make the required changes.

Binbin

On Wed, Jul 26, 2023 at 2:32 PM Logan Dawson ***@***.***>
wrote:

> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> just attaching the
> request I sent via email to this PR.
>
> We need to have your HREF grid2obs stats jobs updated to use BILIN
> interpolation so your stats can be included on the CAM headline plots being
> generated by @MarcelCaron-NOAA <https://github.com/MarcelCaron-NOAA>.
> Right now, your HREF stats are being filtered out because the INTERP_MTHD
> in your stats files is set to NEAREST while other EVS grid2obs stats are
> using BILIN for the INTERP_MTHD.
>
> Will you update your METplus configuration files like below (as
> appropriate)?
>
> {ENSEMBLE/POINT}_STAT_INTERP_TYPE_METHOD = BILIN
> {ENSEMBLE/POINT} _STAT_INTERP_TYPE_WIDTH = 4
>
> You'll also need to update the INTERP_MTHD that's set in the scripts that
> generate your HREF grid2obs.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1652303012>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFLBZMZAUALSTXJJRR3XSFPCVANCNFSM6AAAAAA2XU26A4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/href.ve

BinbinZhou-NOAA committed:
update conf files to set INTERP_TYPE_METHOD = BILIN

BinbinZhou-NOAA commented:
@LoganDawson-NOAA 

   All of the grid2obs conf files have been updated by setting 
{ENSEMBLE/POINT}_STAT_INTERP_TYPE_METHOD = BILIN
{ENSEMBLE/POINT} _STAT_INTERP_TYPE_WIDTH = 4

and all grid2obs plotting scripts have also been updated accordingly.

Binbin

ShelleyMelchior-NOAA commented:
@BinbinZhou-NOAA  -- I will get started on this PR today!

BinbinZhou-NOAA commented:
Shelley,

  I just found an issue in the python plotting scripts after inter_method
was changed to BILIN.
So please hold the href.ve testing until I fix the problem.

Thanks!

Binbin


On Fri, Jul 28, 2023 at 8:58 AM Shelley Melchior ***@***.***>
wrote:

> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> -- I will get
> started on this PR today!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1655645176>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFKYRDUA7EF3RRUNWZTXSOZQFANCNFSM6AAAAAA2XU26A4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Sure thing! Is it OK to get started on the stats testing?

BinbinZhou-NOAA commented:
Shelley,

Yes, stats jobs can be tested right now.

Binbin


On Fri, Jul 28, 2023 at 10:43 AM Shelley Melchior ***@***.***>
wrote:

> Sure thing! Is it OK to get started on the stats testing?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1655809258>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFOQNW7HL3CWVRSZRUDXSPFYRANCNFSM6AAAAAA2XU26A4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/href.ve

BinbinZhou-NOAA committed:
update settings.py

BinbinZhou-NOAA commented:
Shelley,

  The problem in the settings.py has been fixed and updated in the github
href.ve
  Now you can start the plot jobs' testing

Binbin

On Fri, Jul 28, 2023 at 10:45 AM Binbin Zhou - NOAA Affiliate <
***@***.***> wrote:

> Shelley,
>
> Yes, stats jobs can be tested right now.
>
> Binbin
>
>
> On Fri, Jul 28, 2023 at 10:43 AM Shelley Melchior <
> ***@***.***> wrote:
>
>> Sure thing! Is it OK to get started on the stats testing?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1655809258>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQBMPFOQNW7HL3CWVRSZRUDXSPFYRANCNFSM6AAAAAA2XU26A4>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Thanks Binbin! The stats processing is underway. Precip has completed. Grid2obs is still wrapping up (it has been running for 2.5 hours so far). Please review the output.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/cam`
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_precip_stat.o73254172`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_g2o_stat.o73254152`

Can you update `jevs_cam_href_grid2obs_stats.sh` to update line 3 to expand "g2o"?
From
`#PBS -N jevs_cam_href_g2o_stat`
to
`#PBS -N jevs_cam_href_grid2obs_stat`.

Also, in both the grid2obs and precip drivers, you can remove line 1 b/c you have this line in the job cards:
`#PBS -S /bin/bash`

BinbinZhou-NOAA commented:
Shelley,

  All of the output stat files from the precip run are good.
  All of the plots driver script files also have #PBS -S /bin/bash.
Should it be removed from those plots files as well?

Binbin

On Fri, Jul 28, 2023 at 1:22 PM Shelley Melchior ***@***.***>
wrote:

> Thanks Binbin! The stats processing is underway. Precip has completed.
> Grid2obs is still wrapping up (it has been running for 2.5 hours so far).
> Please review the output.
> COMOUT=/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/cam
> logfiles:
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_precip_stat.o73254172
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_g2o_stat.o73254152
>
> Can you update jevs_cam_href_grid2obs_stats.sh to update line 3 to expand
> "g2o"?
> From
> #PBS -N jevs_cam_href_g2o_stat
> to
> #PBS -N jevs_cam_href_grid2obs_stat.
>
> Also, in both the grid2obs and precip drivers, you can remove line 1 b/c
> you have this line in the job cards:
> #PBS -S /bin/bash
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1656045375>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFPCLM4ZC3OII7TIYEDXSPYNDANCNFSM6AAAAAA2XU26A4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Yes! That sounds good, Binbin. I hadn't looked ahead at those yet, so I appreciate the question/effort. Please adjust the plots drivers, too.

BinbinZhou-NOAA committed:
update driver scripts by removing #PBS -S /bin/bash

BinbinZhou-NOAA commented:
Shelley,

  All of the driver scripts have been updated by removing line #PBS -S
/bin/bash
 and #PBS -N jevs_cam_href_g2o_stat has been changed to
  #PBS -N jevs_cam_href_grid2obs_stat
for grid2obs stats driver script file.

Binbin


On Fri, Jul 28, 2023 at 2:00 PM Shelley Melchior ***@***.***>
wrote:

> Yes! That sounds good, Binbin. I hadn't looked ahead at those yet, so I
> appreciate the question/effort. Please adjust the plots drivers, too.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1656113230>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFI625SOR3LYGY4C6GTXSP42ZANCNFSM6AAAAAA2XU26A4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Thanks Binbin. I will start the plots drivers next.
Can you merge auth develop into this feature branch? There has been a recent PR merge for RTOFS.
Also, can you remove these files from your feature branch? This is a good time to clean up related and unused files.
`jevs_href_grid2obs_plots.sh`
`jevs_href_precip_plots.sh`

ShelleyMelchior-NOAA commented:
grid2obs stats finally finished but I think it ran out of walltime. Can you confirm?
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_g2o_stat.o73254152`
Do I need to re-run at a less busy time?

ShelleyMelchior-NOAA commented:
Also, I msiread your previous comment. I wasn't clear enough in my request. For all driver scripts, please remove 
`#!/bin/bash`
but keep
`#PBS -S /bin/bash`

Because of the line in the job cards, you do not need `#!/bin/bash`. This is only true in driver scripts that have job cards. All other scripts still need `#!/bin/bash`.

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/href.ve

ShelleyMelchior-NOAA commented:
plots have completed! See how they look.
COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/cam`
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_*plots.o*`

BinbinZhou-NOAA commented:
Shelley,

  The plot files for all grid2obs tar files are empty, not png files inside.
This means all of grid2obs plots jobs failed.
I checked their output *.o files and can see that the $COMIN is defined
as /lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0
This means the stat files come from the parallel testing directory. However
the grid2obs stat files in the parallel testing directory
are still with interp_method=NEAREST but the new interp_method has been
changed to BILIN.
This caused the string "BILIN" can not be searched in the parallel stat
files by the py script.
 I have changed the interp_method from NEAREST to BILIN in all of my
historical href grid2obs stat files.
You can point the COMIN to my href stat file location
/lfs/h2/emc/vpppg/noscrub/binbin.zhou/evs/v1.0
 to re-run the plots jobs.

Binbin



On Fri, Jul 28, 2023 at 3:41 PM Shelley Melchior ***@***.***>
wrote:

> plots have completed! See how they look.
> COMOUT=/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/cam
> logfiles:
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_*plots.o*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1656230125>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFLIIYJKBPHTTLD44D3XSQIUPANCNFSM6AAAAAA2XU26A4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Got it! That makes sense. I will re-run grid2obs plots pointing COMIN to your space.

BinbinZhou-NOAA committed:
update driver scripts again by adding  #PBS -S /bin/bash

BinbinZhou-NOAA commented:
Shelley,

  The driver scripts have been updated again.
and that 2 useless driver scripts have been deleted.

Binbin

On Fri, Jul 28, 2023 at 3:20 PM Shelley Melchior ***@***.***>
wrote:

> Also, I msiread your previous comment. I wasn't clear enough in my
> request. For all driver scripts, please remove
> #!/bin/bash
> but keep
> #PBS -S /bin/bash
>
> Because of the line in the job cards, you do not need #!/bin/bash. This
> is only true in driver scripts that have job cards. All other scripts still
> need #!/bin/bash.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1656210545>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFOZSBCTPLEKPKFLXHTXSQGHTANCNFSM6AAAAAA2XU26A4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Thanks Binbin! I am re-running the grid2obs plots and things look better! ctc past 90 days is still finishing ...
COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/cam`
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_grid2obs_*plots.o7329*`

BinbinZhou-NOAA commented:
Shelley,

The grid2obs stats were finished normally, not ran out of time. All stat
files are good.
Not need to re-run the this job

Binbin

On Fri, Jul 28, 2023 at 3:17 PM Shelley Melchior ***@***.***>
wrote:

> grid2obs stats finally finished but I think it ran out of walltime. Can
> you confirm?
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_g2o_stat.o73254152
> Do I need to re-run at a less busy time?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1656205438>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFO2HWLLOBO5UL6S3STXSQF27ANCNFSM6AAAAAA2XU26A4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Shelley,

  Did you also re-run the profile plot jobs?

Binbin

On Fri, Jul 28, 2023 at 4:33 PM Shelley Melchior ***@***.***>
wrote:

> Thanks Binbin! I am re-running the grid2obs plots and things look better!
> ctc past 90 days is still finishing ...
> COMOUT=/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/cam
> logfiles:
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_href_grid2obs_*plots.o7329*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1656302317>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFPYP7ZKIT2QMWHUJB3XSQOYFANCNFSM6AAAAAA2XU26A4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Thanks for confirming the grid2obs stats, Binbin!

I hadn't re-run the profile plots, but they are in the queue now, after pointing COMIN to your stats. You can monitor the same COMOUT space noted above. The submission is still in the queue, but you will see log files once it begins to run.

BinbinZhou-NOAA commented:
Shelley,

  Sure I am monitoring the COMOUT directory and can see the plot tar files
if they are completed.
Thanks!

Binbin

On Fri, Jul 28, 2023 at 4:50 PM Shelley Melchior ***@***.***>
wrote:

> Thanks for confirming the grid2obs stats, Binbin!
>
> I hadn't re-run the profile plots, but they are in the queue now, after
> pointing COMIN to your stats. You can monitor the same COMOUT space noted
> above. The submission is still in the queue, but you will see log files
> once it begins to run.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1656321790>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFOYCPLZPSFK4VEYIKLXSQQZRANCNFSM6AAAAAA2XU26A4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/href.ve

BinbinZhou-NOAA commented:
Shelley,

  All of the plot jobs have been completed and all of the plot tar files
in
/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/cam/atmos.20230726
are good.

Thanks!

Binbin

On Fri, Jul 28, 2023 at 4:59 PM Binbin Zhou - NOAA Affiliate <
***@***.***> wrote:

> Shelley,
>
>   Sure I am monitoring the COMOUT directory and can see the plot tar files
> if they are completed.
> Thanks!
>
> Binbin
>
> On Fri, Jul 28, 2023 at 4:50 PM Shelley Melchior ***@***.***>
> wrote:
>
>> Thanks for confirming the grid2obs stats, Binbin!
>>
>> I hadn't re-run the profile plots, but they are in the queue now, after
>> pointing COMIN to your stats. You can monitor the same COMOUT space noted
>> above. The submission is still in the queue, but you will see log files
>> once it begins to run.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/EVS/pull/174#issuecomment-1656321790>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQBMPFOYCPLZPSFK4VEYIKLXSQQZRANCNFSM6AAAAAA2XU26A4>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Excellent! I think this PR is about complete, then. Let me do the final code review to approve the PR for merge.

ShelleyMelchior-NOAA reviewed (approved):
This PR is approved for merge.
NOTE: the PR may not make it into the EVS parallel, though, until Perry returns from vacation, Aug 7.

BinbinZhou-NOAA commented:
Shelley,

  Thank you very much!

Binbin

On Mon, Jul 31, 2023 at 9:09 AM Shelley Melchior ***@***.***>
wrote:

> Merged #174 <https://github.com/NOAA-EMC/EVS/pull/174> into develop.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/174#event-9963743187>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJXUQ4RJGM3U5AWHMDXS6VBVANCNFSM6AAAAAA2XU26A4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683
